src/app/app.ts

```diff
@@ -1,7 +1,10 @@
 declare var Pusher: any;
 
-import {bootstrap} from 'angular2/platform/browser';
-import {Component} from 'angular2/core';
+import {bootstrap} from '@angular/platform-browser-dynamic';
+import {
+  Component,
+  Attribute
```

Comment:
do we need the Attribute import? I can't see where it's used in this file from the diff?